Documentation/user-guides/getting-started.md

```diff
@@ -187,7 +187,7 @@ First, deploy three instances of a simple example application, which listens and
 
 [embedmd]:# (../../example/user-guides/getting-started/example-app-deployment.yaml)
 ```yaml
-apiVersion: extensions/v1beta1
+apiVersion: apps/v1
```

Comment:
when converting to apps/v1 workload versions, you need to specify spec.selector explicitly (it is no longer defaulted from spec.template.metadata.labels). I'd recommend verifying these create successfully

Reply:
Interesting. The Deployment applied just fine on 1.14. I'll add that anyway. Thanks for the hint.